src/components/BalanceDistribution.vue

```diff
@@ -292,16 +322,9 @@ export default defineComponent({
                 width: 100%;
 
                 ::v-deep .trigger {
-                    display: block;
-                }
-
-                ::v-deep .trigger::after {
-                    background: white;
-                }
-
-                ::v-deep .tooltip-box {
-                    background: white;
```

Comment:
Personally, I liked `white` background better. I think it fits better visually. Plus, the dark background clashes with dark-blue identicons: 
![Screenshot from 2022-02-15 10-43-42](https://user-images.githubusercontent.com/1828163/154036525-d227dc14-49f9-4918-a94d-3c19ee0f50f4.png)
@julianbauer What do you think?